spec.html

```diff
@@ -171,15 +171,20 @@ <h1>Example Legacy Normative Optional Clause Heading</h1>
 
 <emu-clause id="sec-normative-references">
   <h1>Normative References</h1>
-  <p>The following referenced documents are indispensable for the application of this document. For dated references, only the edition cited applies. For undated references, the latest edition of the referenced document (including any amendments) applies.</p>
-  <p>ISO/IEC 10646 <i>Information Technology — Universal Multiple-Octet Coded Character Set (UCS) plus Amendment 1:2005, Amendment 2:2006, Amendment 3:2008, and Amendment 4:2008</i>, plus additional amendments and corrigenda, or successor</p>
+  <p>The following referenced documents are indispensable for the application of this document. For undated references, the latest edition of the referenced document (including any amendments) applies.</p>
```

Comment:
Now we have a dated reference and probably should keep the sentence about dated references.